notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    hst_table = Observations.query_criteria(coordinates = ra+dec,radius=".000556 deg", obs_collection="GAIA")
</pre><p>Please add description about why this particular radius was chosen. Also add a line or two examples to increase the radius to capture any star with high proper motion. Also it's always good to show which epoch coordinates you are using.</p><p> </p><p>Also, write why 'GAIA' was chosen for obs_collection.</p><p> </p><p>Finally write that there's no other MAST data for this particular star and yet if there is any, it can be retrieved as written below cells in a new markdown cell. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>I don't know why the version you see is not the current version. This was updated since this. </p>